### PR TITLE
Add health section type filter docs

### PR DESCRIPTION
**Pull request by dwapstra**
Updated docs for health section type filter. Updated Makefile and pinned sphinx version, the latest is incompatible with sphinxcontrib-robotframework.  Removed duplicate make target.

Example of rendered docs with changes:

image

### CODE REVIEW

File: docs/Makefile [hunk 1]
```diff
@@ -92,7 +92,3 @@ html: install_build_deps
 	python -m robot.libdoc genie.libs.robot.GenieRobotApis $(BUILDDIR)/html/userguide/robot_api.html
 	@echo "Build finished. The HTML pages are in $(BUILDDIR)/html."
 
-serve:
```
Comment (tahigash) [inline docs/Makefile:95]: no harm to have?

**Review by tahigash — APPROVED** [review]
approved with 1 comment

File: docs/Makefile (hunk 1 above)
Comment (dwapstra) [reply docs/Makefile:95]: its a duplicate entry

Comment (tahigash) [reply docs/Makefile:95]: not sure duplicate with what. I think this is just launching http server for ease to check.

**Comment by tahigash** [comment]
@dwapstra , removing `serve` is not critical. please find someone else for review since JB will not review soon.

File: docs/Makefile (hunk 1 above)
Comment (dwapstra) [reply docs/Makefile:95]: https://github.com/CiscoTestAutomation/genie/blob/fc61ef0bec6d5a55b9b3bb480bec133a58915229/docs/Makefile#L78

Comment (dwapstra) [reply docs/Makefile:96]: image

Comment (tahigash) [reply docs/Makefile:96]: I didn't notice that. ok. undersood...

**Comment by tahigash** [comment]
@dwapstra , please get one more approval and merge if no issue/concern.

**Comment by tahigash** [comment]
@jeaubin , please review.

**Comment by tahigash** [comment]
@dwapstra any pending on this? please move this forward by getting approval.

**Comment by tahigash** [comment]
@dwapstra please get approve and merge. any pending?

**Review by jeaubin — APPROVED** [review]

**Pull request merged**